heartbeat/IPaddr2

```diff
@@ -449,7 +449,6 @@ ip_init() {
 			exit $OCF_NOT_RUNNING
 		elif [ "$__OCF_ACTION" = stop ]; then
 			ocf_log warn "[$FINDIF] failed"
-			exit $OCF_SUCCESS
```

Comment:
It somehow failed on one of my tests with it, but now it appears to not fail with it, so I'll push an updated commit.